src/react/ISV/components/ISVSteps.tsx

```diff
@@ -0,0 +1,112 @@
+import { spacing, Stepper } from '@scality/core-ui';
+import { useState, createContext, useContext, useMemo } from 'react';
+import { ISVConfiguration } from './ISVConfiguration';
+
+import { ISVConfig, ISVPlatformConfig } from '../types';
+
+import { useTheme } from 'styled-components';
+import { Box } from '@scality/core-ui/dist/next';
+import { ISVSummary } from './ISVSummary';
+import ISVApplyActions from './ISVApplyActions';
+import {
+  VEEAM_BACKUP_REPLICATION_XML_VALUE,
+  VEEAM_OFFICE_365,
+} from '../constants';
+import { useSearchParams } from 'react-router';
+import { Veeam } from '../modules/veeam';
+import { Commvault } from '../modules/commvault';
+import { VeeamVBO } from '../modules/veeam-vbo';
+
+export enum ISVStepsIndexes {
+  Configuration,
+  ApplyActions,
+  Summary,
+}
+
+export type ISVStepperContextType = {
+  platform: ISVPlatformConfig;
+  config: ISVConfig;
+  setConfig: (config: ISVConfig) => void;
+};
+
+export const ISVStepperContext = createContext<
+  ISVStepperContextType | undefined
+>(undefined);
+
+export const useISVStepper = () => {
+  const context = useContext(ISVStepperContext);
+  if (!context) {
+    throw new Error('useISVStepper must be used within ISVStepperProvider');
+  }
+  return context;
+};
+
+export const ISV_STEPS = [
+  {
+    label: 'Configure',
+    Component: ISVConfiguration,
+  },
+  {
+    label: 'Apply Actions',
+    Component: ISVApplyActions,
+  },
+  {
+    label: 'Summary',
+    Component: ISVSummary,
+  },
+] as const;
+
+export const ISVSteps = () => {
+  const theme = useTheme();
+  const [searchParams] = useSearchParams();
+  const id = searchParams.get('platform');
+
+  const getApplication = () => {
```

Comment:
This could be a field of platform so that when we add support for anew app, in theory we can only change the isvlist